orchestrator/settings.py

```diff
@@ -49,6 +49,8 @@ class AppSettings(BaseSettings):
     MAIL_STARTTLS: bool = False
     CACHE_HOST: str = "127.0.0.1"
     CACHE_PORT: int = 6379
+    ENABLE_DISTLOCK_MANAGER: bool = True
+    DISTLOCK_BACKEND: str = "memory"
```

Comment:
Reminder: this also implies a change in the SURF repo: so it will be deployed with REDIS_BACKEND

Reply:
Indeed, will add a note about this in the issue!